manifests/i/IndexEducation/ClientPRONOTE2024-64bit/3.11.0/IndexEducation.ClientPRONOTE2024-64bit.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: IndexEducation.ClientPRONOTE2024-64bit
+PackageVersion: 3.11.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: --exact --accept-package-agreements --accept-source-agreement --silent flag --force
+  SilentWithProgress: --exact --accept-package-agreements --accept-source-agreement --silent flag --force

```

```suggestion
  Silent: /S
```

For this package, I still believe that `/S` (or other parameters, rather than WinGet parameters) should be used as the silent installation switch.